webui/react/src/pages/TaskList.tsx

```diff
@@ -190,24 +267,100 @@ const TaskList: React.FC = () => {
       <TaskActionDropdown task={record} onComplete={handleActionComplete} />
     );
 
-    return [ ...defaultColumns ].map(column => {
+    const tableColumns: ColumnType<CommandTask>[] = [
+      {
+        dataIndex: 'id',
+        key: 'id',
+        render: taskIdRenderer,
+        sorter: (a: CommandTask, b: CommandTask): number => alphanumericSorter(a.id, b.id),
+        title: 'Short ID',
+      },
+      {
+        filterDropdown: typeFilterDropdown,
+        filters: Object.values(CommandType)
```

Comment:
What about adding type icons here?

<img width="203" alt="Schermata 2021-06-01 alle 08 21 25" src="https://user-images.githubusercontent.com/5413702/120276031-75210e80-c2b2-11eb-8013-0e006b34aedf.png">

Reply:
great idea

Reply:
Ended up having icons and labels together. With just the icons, it looked sparse and the filter dropdown had a minimum size required to house both the `Reset` and `Ok` buttons.